alert/datapoint.go

```diff
@@ -0,0 +1,12 @@
+package alert
+
+import (
+	"time"
+)
+
+// A MetricRecord is a recording of an Alert metric.
+type MetricRecord struct {
+	AlertID   int       `json:"alert_id"`
+	ServiceID string    `json:"service_id"`
+	ClosedAt  time.Time `json:"closed_at"`
+}
```

Comment:
Can this go in the `alertmetrics` package as a `Record` type instead? Also, the `json` tags can be removed